app/monitoringapi.go

```diff
@@ -90,7 +91,15 @@ func wireMonitoringAPI(ctx context.Context, life *lifecycle.Manager, addr string
 		ReadHeaderTimeout: time.Second,
 	}
 
+	// Create and start health checker.
+	checker := health.NewChecker(health.Metadata{
+		NumValidators: len(pubkeys),
+		NumPeers:      len(peerIDs),
+		QuorumPeers:   cluster.Threshold(len(peerIDs)),
```

Comment:
The `threshold` value should be a function parameter in `wireMonitoringAPI` which app passes `int(cluster.Threshold)` as value